CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Next Release
 
+*
+
+## v0.3.10 (February 25, 2025)
+
 * Fix Colab environment detection logic ([#230](https://github.com/Kaggle/kagglehub/pull/230))
 
 ## v0.3.9 (February 18, 2025)
```

Comment:
unrelated but noticing we missed the Package feature here :) unless that was intentional cause we were pre-launch?

Reply:
let me add it.